### PR TITLE
feat : a card component and refactored style issues

### PR DESCRIPTION
**Pull request by ShekharDhangar**
Added Vertical Card component. Refactored style file. Refer screenshot attached.
image

### CODE REVIEW

**Review by animeshgarai09 — APPROVED** [review]
All looks good👍🏻

File: components/Navigation/navigation.css [hunk 1]
```diff
@@ -95,7 +97,7 @@
 .sidebar{
     min-width: 15vw;
     height: 100%;
-    background-color: white;
+    background-color: var(--clr-white);
```
Comment (Hamza-777) [inline components/Navigation/navigation.css:100]: better variable naming can be done. like --color-white.

File: utils.css [hunk 2]
```diff
@@ -2,19 +2,19 @@
 :root{
     --primary-clr:#05386b;
     --secondary-clr:#04478a;
-    --danger-clr: #FF0000;
-    --warning-clr:rgb(234 233 12);
-    --success-clr:#0ebf28;
+    --danger-clr: #ed3535;
+    --warning-clr:#c1c1c0;
+    --success-clr:#26a338;
```
Comment (Hamza-777) [inline utils.css:7]: Consistent naming like either --color-colorname or --colorname-color throughout will make it easier for you and everyone else as well.

**Review by Hamza-777 — APPROVED** [review]

**Comment by Hamza-777** [comment]
Everything else looks awesome.

File: utils.css (hunk 2 above)
Comment (ShekharDhangar) [reply utils.css:7]: Yes, that I thought, but its just personal preference to use `clr` instead of `color`. I hope it justifies.

**Pull request merged**